docs/commerce/order_management/order_management.md

```diff
@@ -0,0 +1,18 @@
+---
+description: Order management module handles orders and allows managing orders in the system.
+edition: commerce
+---
+
+# Order management
+
+The Order management module allows the system to create orders following a purchase made by a customer of the e-commerce presence.
+
+With the Back Office **Orders** screen, you can search for orders and filter search results. 
+Depending on the permissions assigned to your [user role](permissions_and_users.md), you might be able to review order details, track order fulfillment status, and cancel orders.
```

Comment:
```suggestion
Depending on the permissions assigned to your [user role](permissions_and_users.md), you might be able to review order details, track order completion status, and cancel orders.
```
just because "complete" is used in the interface, to keep consistent naming